src/plugins/vis_type_timeseries/public/application/components/vis_with_splits.js

```diff
@@ -105,7 +104,12 @@ export function visWithSplits(WrappedComponent) {
         },
       };
       return (
-        <div key={key} className="tvbSplitVis__split">
+        <div
+          key={key}
+          className={
+            visData[model.id].series.length === 1 ? 'tvbSplitVis__split__one' : 'tvbSplitVis__split'
+          }
+        >
```

Comment:
And so you don't need this new class either.

```suggestion
        <div key={key} className="tvbSplitVis__split">
```

Reply:
done